nginx/default.conf

```diff
@@ -0,0 +1,36 @@
+server {
+  listen 8080;
+
+  location /api/user-service {
+    proxy_pass http://user-service:8000/api/user;
```

Comment:
Have you tested if the JWT token can be sent through this reverse proxy for authentication? If it works, ignore this comment.

Reply:
Yup can